packages/block-editor/src/components/color-signal-slot-fill/index.js

```diff
@@ -0,0 +1,24 @@
+import {
+  createSlotFill,
+  __experimentalUseSlot as useSlot
+} from '@wordpress/components';
+
+const slotName = 'NovaBlocksColorSignal'
+
+const {Fill, Slot: NovaBlocksColorSignalSlot } = createSlotFill( slotName );
```

Comment:
You can do 
```js
const ColorSignalCustomizeControls = createSlotFill( slotName );
```
and than use and export the `Slot` and `Fill` properties of that object